### PR TITLE
Update record syntax in examples

### PR DESCRIPTION
**Pull request by nilslindemann**

### CODE REVIEW

File: core/source/Dom.mint [hunk 1]
```diff
@@ -180,14 +180,14 @@ module Dom {
...
-      x = 0,
-      y = 0
+      bottom : 0,
+      height : 0,
+      width : 0,
+      right : 0,
+      left : 0,
+      top : 0,
+      x : 0,
+      y : 0
```
Comment (Sija) [inline core/source/Dom.mint:190]:
```suggestion
      bottom: 0,
      height: 0,
      width: 0,
      right: 0,
      left: 0,
      top: 0,
      x: 0,
      y: 0
```

File: core/source/WebSocket.mint [hunk 2]
```diff
@@ -41,12 +41,12 @@ module WebSocket {
...
-        onMessage = handleMessage,
-        onError = handleError,
-        onClose = handleClose,
-        onOpen = handleOpen
+        url : "wss://echo.websocket.org",
+        reconnectOnClose : true,
+        onMessage : handleMessage,
+        onError : handleError,
+        onClose : handleClose,
+        onOpen : handleOpen
```
Comment (Sija) [inline core/source/WebSocket.mint:49]:
```suggestion
        url: "wss://echo.websocket.org",
        reconnectOnClose: true,
        onMessage: handleMessage,
        onError: handleError,
        onClose: handleClose,
        onOpen: handleOpen
```

**Review by Sija — CHANGES_REQUESTED** [review]

**Review by Sija — APPROVED** [review]

**Comment by nilslindemann** [comment]
Thanks, Sija

**Comment by nilslindemann** [comment]
Thx

**Pull request merged**